plugins/obs-webrtc/whip-output.cpp

```diff
@@ -5,16 +5,19 @@ const char signaling_media_id_valid_char[] = "0123456789"
 					     "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 					     "abcdefghijklmnopqrstuvwxyz";
 
-const uint32_t audio_ssrc = 5002;
+const uint32_t audio_ssrc = generate_random_u32();
 const char *audio_mid = "0";
 const uint32_t audio_clockrate = 48000;
 const uint8_t audio_payload_type = 111;
 
-const uint32_t video_ssrc = 5000;
+const uint32_t video_ssrc = generate_random_u32();
 const char *video_mid = "1";
 const uint32_t video_clockrate = 90000;
 const uint8_t video_payload_type = 96;
 
+// user-agent to use for requests constructed at Init()
```

Comment:
This comment looks obvious. Is this necessary?

Reply:
I'd say no; sometimes I forget to remove "notes to self"